challenge_7/python/ajschrier/README.md

```diff
@@ -0,0 +1,19 @@
+# Challenge 7 - Find the Missing Number
+
+In a list 0..N, find which digit is missing.
+
+# Functions
+
+## builtInMethod
+
+**Input:** List
+**Output:** Integer
+
+Finds the integer 0 < x < N not present in list. The 'not in' syntax performs at O(N) according to [the Python Wiki](https://wiki.python.org/moin/TimeComplexity), so that is used to check the list for the value in the range. 
```

Comment:
By utilizing it in your xrange loop you are creating a O(n^2) solution. The first O(n) is your loop, which is multiplied by the nested O(n) of the `in` operator.